packages/gatsby-recipes/recipes/travis-deploy-github-pages.mdx

```diff
@@ -0,0 +1,28 @@
+# Deploy to Github Pages with travis-ci
+
+Github Pages is a solution to host static websites directly in your Github repository by pushing site files to `gh-pages` branch.
+
+---
+
+Define basic travis-ci configuration to build and push you site on `gh-pages` page on each new commit on master.
+
+<File
+  path=".travis.yml"
+  content="https://gist.githubusercontent.com/Slashgear/a52090c90e672e129d1bf8957785d34f/raw/5ede57769a21084748784da3749ee4cadef5cd0a/.travis-ci.yml"
+/>
+
+---
+
+Generate a Github token on [this page](https://github.com/settings/tokens/new).
+
+---
+
+Toggle on your repository in [your travis-ci configuration page](https://travis-ci.org/account/repositories).
+
+---
+
+Open Travis setting for this repository and add the secret _Environnement variable_ **GITHUB_TOKEN** with your previously generated token.
```

Comment:
Should we add the url here, too?

```suggestion
Open Travis settings for this repository and add the secret _Environnement variable_ **GITHUB_TOKEN** with your previously generated token.
```

Reply:
Url depends on `repository` name and "user".
Sadly we could not get _a gatsby app context_ for now with recipes for now.